lambdas/validate-oauth-callback/src/main/java/uk/gov/di/ipv/core/validateoauthcallback/ValidateOAuthCallbackHandler.java

```diff
@@ -57,6 +59,7 @@
                     OAuth2Error.TEMPORARILY_UNAVAILABLE_CODE);
     private static final String PYI_TECHNICAL_UNRECOVERABLE_ERROR_PAGE_ID =
             "pyi-technical-unrecoverable";
+    private static final String PYI_ATTEMPT_RECOVERY_PAGE_ID = "attempt-recovery";
```

Comment:
Not sure yet what the actual page id will be, but should it have pyi- at the start to match others?

Reply:
the page makes sense to be called `pyi-attempt-recovery`

Reply:
Updated this